physics/smoke_dust/rrfs_smoke_wrapper.meta

```diff
@@ -747,6 +769,43 @@
   type = real
   kind = kind_phys
   intent = inout
+[mpicomm]
+  standard_name = mpi_communicator
+  long_name = MPI communicator
+  units = index
+  dimensions = ()
+  type = integer
+  intent = in
+[mpirank]
+  standard_name = mpi_rank
+  long_name = current MPI-rank
+  units = index
+  dimensions = ()
+  type = integer
+  intent = in
+[mpiroot]
+  standard_name = mpi_root
+  long_name = master MPI-rank
+  units = index
+  dimensions = ()
+  type = integer
+  intent = in
+[uspdavg]
+  standard_name = bl_averaged_wind_speed
+  long_name = average wind speed within the boundary layer
+  units = none
```

Comment:
Is the unit `m s-1`?

Reply:
@grant. Yes, the unit will be updated to m s-1. Thanks.